cmd/deploy/deploy.go

```diff
@@ -641,6 +642,11 @@ func (dc *Command) deployDependencies(ctx context.Context, deployOptions *Option
 	for depName, dep := range deployOptions.Manifest.Dependencies {
 		oktetoLog.Information("Deploying dependency  '%s'", depName)
 		oktetoLog.SetStage(fmt.Sprintf("Deploying dependency %s", depName))
+
+		if err := validator.CheckReservedEnvName(dep.Variables); err != nil {
```

Comment:
```suggestion
		if err := validator.CheckReservedVarName(dep.Variables); err != nil {
```

Reply:
nit. I find misleading `EnvName`,, maybe `VarName` is more explicit?

Reply:
i used this because of the type.... in this case is not a list of strings, but an env.Env 🤷🏻‍♀️

Reply:
@teresaromero the type is `env.Var`

Reply:
nothing, i was wrong. its already changed